server/tests/spec_helper.js

```diff
@@ -0,0 +1,12 @@
+import mongoose from 'mongoose';
+import chai from 'chai';
+
+chai.config.includeStack = true;
+global.expect = chai.expect;
+
+afterEach((done) => {
+  mongoose.connection.db.dropDatabase();
+  done();
```

Comment:
You might want to put this in the callback for the line above so that it waits for the DB to be completely dropped before starting the next test